app/styles/module/_ads.scss

```diff
@@ -205,15 +205,29 @@ $closeButtonHeight: 40px;
   }
 }
 
-.new-logo {
-  animation-duration: 3s;
-  animation-fill-mode: forwards;
-  animation-name: fade-in;
-  max-height: 55px;
-  opacity: 0;
-  padding-left: 12px;
-}
+@mixin logo-replacement($max-height, $padding-left) {
```

Comment:
This mixin is different than the one created in ad-engine and I wonder if in the sake of consistency we shouldn't make them accept the same params 🤔 https://github.com/Wikia/ad-engine/pull/948/files#diff-20e570b6d6f5f0fe51f509457857db4705fb7388b2f8a38ded22154a6ddc039cR61

Reply:
Discussed on a call. Let's leave it as-is.